qemu/tests/watchdog.py

```diff
@@ -282,6 +286,26 @@ def hotplug_unplug_watchdog_device():
                 test.fail("Oops, find watchdog pci, unplug failed")
             logging.info("The WDT remove successfully")
 
+    def watchdog_test_suit():
+        """
+        Run watchdog-test-framework to verify the function of emulated watchdog
+        devices.
+        """
+
+        _watchdog_device_check(test, session, watchdog_device_type)
+        test_lib = params["test_lib"]
+        test_lib_path = os.path.join(data_dir.get_deps_dir(), test_lib)
+        vm.copy_files_to(test_lib_path, "/home")
```

Comment:
What about use `data_dir.get_tmp_dir()` instead of `/home`. There it will be cleaned by framework after test finished.

Reply:
It can only be used on host.